CONTRIBUTING.md

```diff
@@ -95,14 +97,14 @@ determined by the headers included in the source files and headers of
 those libraries and executables) in the `CMakeLists.txt` files'
 `TARGET_LINK_LIBRARIES` calls, and omit any transitive dependencies.
 
-For example, the `dem_2d` application
-[links to `lethe-core` and `lethe-dem`](applications/dem_2d/CMakeLists.txt)
+For example, the `lethe-particles` application
+[links to `lethe-core` and `lethe-dem`](applications/lethe-particles/CMakeLists.txt)
 because it
-[includes core and DEM headers](applications/dem_2d/dem_2d.cc),
-but the `gd_navier_stokes_2d` application
-[links only to `lethe-solvers`](applications/gd_navier_stokes_2d/CMakeLists.txt)
+[includes core and DEM headers](applications/lethe-particles/dem_2d.cc),
+but the `lethe-fluid` application
+[links only to `lethe-solvers`](applications/lethe-fluid/CMakeLists.txt)
 because it
-[includes only solvers headers](applications/gd_navier_stokes_2d/gd_navier_stokes_2d.cc),
+[includes only solvers headers](applications/lethe-fluid/gd_navier_stokes_2d.cc),
```

Comment:
We should put the name of the executable as they are now here. dem_2d and gd_navier_stokes_2d do not exist anymore. :)